node/sc/bridge_manager.go

```diff
@@ -270,26 +271,41 @@ func (bi *BridgeInfo) UpdateInfo() error {
 	return nil
 }
 
-// handleRequestValueTransferEvent handles the given request value transfer event.
-func (bi *BridgeInfo) handleRequestValueTransferEvent(ev *RequestValueTransferEvent) error {
-	tokenType := ev.TokenType
-	tokenAddr := bi.GetCounterPartToken(ev.TokenAddress)
-	// TODO-Klaytn-Servicechain Add counterpart token address in requestValueTransferEvent
-	if tokenType != KLAY && tokenAddr == (common.Address{}) {
-		logger.Warn("Unregistered counter part token address.", "addr", tokenAddr.Hex())
-		ctTokenAddr, err := bi.counterpartBridge.RegisteredTokens(nil, ev.TokenAddress)
```

Comment:
Okay you removed the runtime token registering logic, and just return error when something goes wrong. But is it sure this error does not occur when we fix initial token register logic? I think we need to do some test for this.

Reply:
The returns that return errors are actually fatal errors, meaning that the handle value transfer for the two types of the token must not be correctly handled. Please compare the previous code and this new change. You can find the key difference that a handle token address was retrieved from where. The handle token address was retrieved by asking for the counterpart chain's contract, not the own chain's contract. It opens a gateway to be victim's chain handles unintended value transfer events, although the unknown token contract has not been registered before.